tools/ci_build/github/azure-pipelines/stages/py-gpu-packaging-stage.yml

```diff
@@ -79,6 +79,6 @@ stages:
           MACHINE_POOL: 'onnxruntime-Win2022-GPU-dml-A10'
           PYTHON_VERSION: ${{ python_version }}
           EP_BUILD_FLAGS: --use_dml --cmake_extra_defines CMAKE_SYSTEM_VERSION=10.0.18362.0 --enable_wcos
-          ENV_SETUP_SCRIPT: setup_env.bat
           EP_NAME: directml
-          cmake_build_type: ${{ parameters.cmake_build_type }}
\ No newline at end of file
+          cmake_build_type: ${{ parameters.cmake_build_type }}
+          CudaVersion: ${{ parameters.cuda_version }}
```

Comment:
We should not need to set CUDA version when the EP name is DML?

Reply:
That is correct. We can just use default.